include/mvd/bits/mvd3_misc.hpp

```diff
@@ -104,7 +105,10 @@ const std::string did_lib_circuit_seeds = "/circuit/seeds";
 
 namespace MVD3 {
 
-inline MVD3File::MVD3File(const std::string &str) : _filename(str), _hdf5_file(str), _nb_neurons(0)
+inline MVD3File::MVD3File(const std::string &str) : _filename(str), _hdf5_file(str), _tsv_file(nullptr), _nb_neurons(0)
```

Comment:
Since you changed this line apply latest code conventions, i.e.
```suggestion
inline MVD3File::MVD3File(const std::string &str)
    : _filename(str)
    , _hdf5_file(str)
    , _tsv_file(nullptr)
    , _nb_neurons(0)
```